tests/Unit/Visualization/Python/Test_PlotTrajectories.py

```diff
@@ -0,0 +1,143 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+import shutil
+import unittest
+
+import numpy as np
+from click.testing import CliRunner
+
+import spectre.IO.H5 as spectre_h5
+from spectre.Informer import unit_test_build_path, unit_test_src_path
+from spectre.Visualization.PlotTrajectories import plot_trajectories_command
+
+
+class TestPlotTrajectories(unittest.TestCase):
```

Comment:
Add this test to the `CMakeLists.txt` file in this directory, so it runs automatically on GitHub.